mod-di-converter-storage-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -305,7 +305,7 @@
     },
     {
       "tableName": "metadata_internal",
-      "fromModuleVersion": "mod-di-converter-storage-2.3.0-SNAPSHOT",
```

Comment:
@KaterynaSenchenko is it okay if we use this version in Q release?

Reply:
for Q we'll need to set 2.2.1

Reply:
different versions on different release branches, for Ramsons (master) we should probably leave the Q version

Reply:
ok, @KaterynaSenchenko  should I return the snapshot version for this master branch, I mean 2.3.0-SNAPSHOT

Reply:
Let's set 2.2.1 for master

Reply:
Got it. Thank you!

Reply:
maybe we need to set 2.2.0 (or 2.1.8) version since this table has been already released

Reply:
uh, it's the wrong script. 2.2.0 here - Q released version